hooks/useNewSarsCov2Data.tsx

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from "@tanstack/react-query";
+import { gql } from "@apollo/client";
+import { request } from 'graphql-request';
+import { SarsCov2EstimatesQuery } from "@/gql/graphql";
+
+export const sarsCov2Esimates = gql`
```

Comment:
```suggestion
export const sarsCov2Estimates = gql`
```
Whoops, I'm a silly goose. I fix this later.